docs/content/release-notes/flink-1.14.md

```diff
@@ -0,0 +1,422 @@
+for all edges of the stream graph. A new option `execution.batch-shuffle-mode` allows to change it
+to pipelined behavior if necessary.
+
+#### Allow @TypeInfo annotation on POJO field declarations
+
+##### [FLINK-12141](https://issues.apache.org/jira/browse/FLINK-12141)
+
+`@TypeInfo` annotations can now also be used on POJO fields which, for example, can help to define
+custom serializers for third-party classes that can otherwise not be annotated themselves.
+
+### Table & SQL
+
+#### Use pipeline name consistently across DataStream API and Table API
+
+##### [FLINK-23646](https://issues.apache.org/jira/browse/FLINK-23646)
+
+The default job name for DataStream API programs in batch mode has changed from `"Flink Streaming Job"` to
+`"Flink Batch Job"`. A custom name can be set with config option `pipeline.name`.
+
+#### Propagate unique keys for fromChangelogStream
+
+##### [FLINK-24033](https://issues.apache.org/jira/browse/FLINK-24033)
+
+Compared to 1.13.2, `StreamTableEnvironment.fromChangelogStream` might produce a different stream
+because primary keys were not properly considered before.
+
+#### Support new type inference for Table#flatMap
+
+##### [FLINK-16769](https://issues.apache.org/jira/browse/FLINK-16769)
+
+`Table.flatMap()` supports the new type system now. Users are requested to upgrade their functions.
+
+#### Add Scala implicit conversions for new API methods
+
+##### [FLINK-22590](https://issues.apache.org/jira/browse/FLINK-22590)
+
+The Scala implicits that convert between DataStream API and Table API have been updated to the new
+methods of FLIP-136.
+
+The changes might require an update of pipelines that used `toTable` or implicit conversions from
+`Table` to `DataStream[Row]`.
+
+#### Remove YAML environment file support in SQL Client
+
+##### [FLINK-22540](https://issues.apache.org/jira/browse/FLINK-22540)
+
+The sql-client-defaults.yaml YAML file was deprecated in 1.13 release and now it is totally removed
+in this release. As an alternative, you can use -i startup option to execute an initialization SQL
+file to setup the SQL Client session. The so-called initialization SQL file can use Flink DDLs to
+define available catalogs, table sources and sinks, user-defined functions, and other properties
+required for execution and deployment.
+
+See more: https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sqlclient/#initialize-session-using-sql-files
+
+#### Remove the legacy planner code base
+
+##### [FLINK-22864](https://issues.apache.org/jira/browse/FLINK-22864)
+
+The old Table/SQL planner has been removed. BatchTableEnvironment and DataSet API interop with Table
+API are not supported anymore. Use the unified TableEnvironment for batch and stream processing with
+the new planner or the DataStream API in batch execution mode.\r\n\r\nUsers are encouraged to update
+their pipelines. Otherwise Flink 1.13 is the last version that offers the old functionality.
```

Comment:
does \r\n render properly?
```suggestion
the new planner or the DataStream API in batch execution mode.

Users are encouraged to update
their pipelines. Otherwise Flink 1.13 is the last version that offers the old functionality.
```

Reply:
No, it does not. I must've missed this occurence.